coco/master.py

```diff
@@ -165,12 +223,14 @@ def _load_config(self, config_path):
 
         self.log_level = config.get("log_level", "INFO")
         self.endpoint_dir = config["endpoint_dir"]
+        self.prom_dir = config["prometheus_multiproc_dir"]
```

Comment:
Can we create a tmp dir instead to reduce config vars?
https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory